spec.md

```diff
@@ -85,7 +85,7 @@ The `Span` interface must have the following capabilities:
 
 - Finish the (already-started) `Span`.  Finish should be the last call made to any span instance, and to do otherwise leads to undefined behavior. **(py: `finish`, go: `Finish`)** Some implementations may record information about active `Span`s before they are `Finish`ed (e.g., for long-lived `Span` instances), or `Finish` may never be called due to host process failure or programming errors. Implementations should clearly document the `Span` durability guarantees they provide in such cases.
 - Set a key:value tag on the `Span`. The key must be a `string`, and the value must be either a `string`, a `boolean`, or a numeric type. Behavior for other value types is undefined. If multiple values are set to the same key (i.e., in multiple calls), implementation behavior is also undefined. **(py: `set_tag`, go: `SetTag`)**
-- Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). An optional timestamp can be used to specify a past timestamp. **(py: `log`, go: `Log`)**
+- Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). An optional timestamp can be used to specify a past timestamp. An implementation may choose to serialize log payloads asynchronously, to avoid blocking the caller with potentially expensive computation; callers should avoid mutating log payloads after they are logged or pre-serialize log payloads. **(py: `log`, go: `Log`)**
```

Comment:
LGTM, though I'd like to give other folks a few days to chime in before merging.